### ISSUE TITLE
Elevated circulating uracil concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6033]
**Preferred term label:** Elevated circulating uracil concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of uracil in the blood is above the normal range., PMID: 3910670

**Parent term (use hpo.jax.org/app)** Abnormal circulating nitrogen compound concentration HP:0004364

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
See this disease: https://omim.org/entry/222748

**Comment by pnrobinson** [comment]
added as HP:0033139
